pandas/sparse/tests/test_arithmetics.py

```diff
@@ -281,28 +281,29 @@ def test_int_array(self):
             self._check_numeric_ops(a, b, values, rvalues)
 
     def test_int_array_comparison(self):
-        values = self._base([0, 1, 2, 0, 0, 0, 1, 2, 1, 0])
-        rvalues = self._base([2, 0, 2, 3, 0, 0, 1, 5, 2, 0])
 
-        dtype = np.int64
+        # int32 NI ATM
```

Comment:
@sinhrks so this fixes for windows currently. when support int32 can add tests here.